pkg/database/realm.go

```diff
@@ -288,6 +288,10 @@ type Realm struct {
 	// value is to allow all test types.
 	AllowedTestTypes TestType `gorm:"type:smallint; not null; default: 14;"`
 
+	// AllowUserReportWebView - if enabled, will use the user report web view
+	// on the redirect server for this realm. If disabled, it will 404.
+	AllowUserReportWebView bool
```

Comment:
Still need to set all the `gorm` properties because of automigrate. Also need to do it for `AllowAdminUserReport` (I missed that earlier).